lib/nimble_template/addons/variants/phoenix/web/assets.ex

```diff
@@ -61,4 +62,18 @@ defmodule NimbleTemplate.Addons.Phoenix.Web.Assets do
 
     project
   end
+
+  defp gzip_plug_static(%Project{web_path: web_path} = project) do
```

Comment:
How about `enable_gzip_for_static_assets` 💡

Reply:
Thanks Ros, changed in https://github.com/nimblehq/elixir-templates/pull/150/commits/17556f1693ae7fdfa75be47ccea121823c0de6d9 :D